### DISCUSSION TITLE
Manage and Reorder Tabs

### DISCUSSION THREAD

**Discussion by DOM-LGH** [MissingCore/Music#125] [Ideas]
Rearrange the order of tabs - Album, Playlist, Artist etc. I am a user who doesn't always utilize the "Folder" tab.

**Comment by cyanChill** [comment]
This isn't a bad idea, though it'll probably come in a future update. The next major update includes swipe navigation on the home screen, which should make navigating easier while you wait for this to be implemented.

**Comment by cyanChill** [comment]
If you're using the APK version of the app, you can try this out in the [`v2.1.0-rc.2`](https://github.com/MissingCore/Music/releases/tag/v2.1.0-rc.2) release. We added the ability to change the order and visibility of the tabs after the "Home" tab (hiding tabs might cause some unexpected behaviors to occur and remove some features).

**Comment by DOM-LGH** [comment]
Thank you for reaching out! I have downloaded and installed the APK and
this is a welcome feature. I also notice the UI updated and more
customization options!

On Fri, Jan 24, 2025 at 3:10 PM Anthony Liang ***@***.***>
wrote:

> If you're using the APK version of the app, you can try this out in the
> v2.1.0-rc.2
> <https://github.com/MissingCore/Music/releases/tag/v2.1.0-rc.2> release.
> We added the ability to change the order and visibility of the tabs after
> the "Home" tab (hiding tabs might cause some unexpected behaviors to occur
> and remove some features).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/MissingCore/Music/discussions/125#discussioncomment-11947206>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BN52TPBJYNPRUI64ELQTIQ32MKM3TAVCNFSM6AAAAABUDZEBCCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCOJUG4ZDANQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thanks,
Dominic
